GameData/RP-1/Contracts/Commercial Applications 1/EarlyComSat-CA.cfg

```diff
@@ -133,9 +133,9 @@ CONTRACT_TYPE
 			name = Orbit
 			type = Orbit
 			minInclination = Round(Random(30, 31))
-			minPeA = Round(Random(700000, 1000000), 1000)
-			minApA = Round(Random(3000000, 10000000), 1000)
-			maxApA = Round(Random(10000000, 30000000), 1000)
+			minPeA = Round(Random(700000, 1000000), 10000)
+			minApA = Round(Random(3000000, 10000000), 10000)
```

Comment:
This still feels like a ton of variation? My hunch is we might not actually want variation here, per other limit-variation stuff @ppboyle has done. If there is variation, and since there's no decline penalty, the player can just keep declining until they get what they want. If we want variance we could vary things like LAN, or payload (using the payload scaling logic in the old GEO repeatable?)...not sure what else?

Reply:
Funny enough that was lifted straight out of the original EarlyComSat.  This is in the geo repeatable: payload = RP1CommsPayload().  I added back in the random payload from the original, but I'm fine with whichever on it. Tightening up the orbit variation but adding in a random LAN does sound like a good change to it, though.

Reply:
Yeah I think Max only hit some of the old commsat contracts, not all? Might have been we punted until the revamp anyway, which is...now :D